samples/flattened_theia-next.yaml

```diff
@@ -37,60 +37,11 @@ spec:
-                              # with che-theia.eclipse.org/vscode-extensions attributes to get VSX urls
-                              # Those found in the dedicated containers components are with a sidecar,
-                              # Those found in the che-theia container are without a sidecar.
-        attributes:
-          "app.kubernetes.io/part-of": che-theia.eclipse.org
-          "app.kubernetes.io/component": bootstrapper
-        container:
-          args:
-            - /bin/sh
-            - '-c'
-            - |
-              KUBE_API_ENDPOINT="https://kubernetes.default.svc/apis/workspace.devfile.io/v1alpha2/namespaces/${CHE_WORKSPACE_NAMESPACE}/devworkspaces/${CHE_WORKSPACE_NAME}" &&\
-              TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) &&\
-              WORKSPACE=$(curl -fsS --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer ${TOKEN}" $KUBE_API_ENDPOINT) &&\
-              for container in $(echo $WORKSPACE | sed -e 's;[[,]\({"attributes":{"app.kubernetes.io\);\n\1;g' | grep '"che-theia.eclipse.org/vscode-extensions":' | grep -e '^{"attributes".*'); do \
-                dest=$(echo "$container" | sed 's;.*{"name":"THEIA_PLUGINS","value":"local-dir://\([^"][^"]*\)"}.*;\1;' - ) ;\
-                urls=$(echo "$container" | sed 's;.*"che-theia.eclipse.org/vscode-extensions":\[\([^]][^]]*\)\]}.*;\1;' - ) ;\
-                mkdir -p $dest ;\
-                for url in $(echo $urls | sed 's/[",]/ /g' - ); do \
-                  echo; echo downloading $urls to $dest; curl -L $url > $dest/$(basename $url) ;\
-                done \
-              done \
-          image: 'quay.io/samsahai/curl:latest'
-          volumeMounts:
-            - path: "/plugins"
-              name: plugins
-      - name: remote-runtime-injector
-        attributes:
-          "app.kubernetes.io/part-of": che-theia.eclipse.org
-          "app.kubernetes.io/component": bootstrapper
-        container:                          #### corresponds to `initContainer` definition in old meta.yaml.
-          image: "quay.io/eclipse/che-theia-endpoint-runtime-binary:7.20.0"
-          volumeMounts:
-            - path: "/remote-endpoint"
-              name: remote-endpoint
-          env:
-            - name: PLUGIN_REMOTE_ENDPOINT_EXECUTABLE
-              value: /remote-endpoint/plugin-remote-endpoint
-            - name: REMOTE_ENDPOINT_VOLUME_NAME
-              value: remote-endpoint
       - name: theia-ide
         attributes:
           "app.kubernetes.io/name": che-theia.eclipse.org
           "app.kubernetes.io/part-of": che.eclipse.org
           "app.kubernetes.io/component": editor
-
-          # Added by Che-theia at start when detecting, after cloning, that the extensions.json in the repo
-          # contains the vscode-pull-request-github vscode plugin.
-          "che-theia.eclipse.org/vscode-extensions":
```

Comment:
plugins are removed from simple theia-next sample, since it fails for me on minikube even from https://github.com/devfile/devworkspace-operator/pull/286 with error:
```
downloading "https://github.com/microsoft/vscode-pull-request-github/releases/download/v0.8.0/vscode-pull-request-github-0.8.0.vsix" to /plugins
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   645  100   645    0     0   4640      0 --:--:-- --:--:-- --:--:--  4640
100  468k  100  468k    0     0   960k      0 --:--:-- --:--:-- --:--:-- 5002k
mkdir: can't create directory 'for': Permission denied

downloading for to for
/bin/sh: can't create for/for: nonexistent directory
mkdir: can't create directory 'deployment': Permission denied
```
So, I just don't won't to invest time in that vsinstaller, but it may be a sing that something is wrong with PVC on DevWorkspace Operator level.

Reply:
:thinking: I'm 90% sure the issue should be fixed (the problem was that we do a for loop over a big string and by default bash splits on `' '`. But I agree, we can remove vsx installer.

Reply:
It's fixed now. But I still removing vsx installer from Theia Next to keep to simple.